README.md

```diff
@@ -42,10 +43,16 @@ source thirdparty/set_thirdparty_env.sh
 ```
 
 The provided script `setup_build_env.sh` sets up a build environment for you
-with third party dependencies.  You use it by running `source
-setup_build_env.sh`.  By default, it will create a build directory `build/`.
-You can override the build directory by setting the BUILD_DIR env variable to
-another location.
+with third party dependencies.  You use it by running
+
+```shell
+source setup_build_env.sh
+```
+
+(You can't build Thrift on macOS with this script! Use Homebrew.)
```

Comment:
I would also prefer to move the single-line Linux version above the three-line sample of what to run. I agree that it's a lot easier to read, "paste this for Mac and paste this for Linux".